src/tezos_interop/tezos_bridge.mli

```diff
@@ -1,11 +1,22 @@
 open Crypto
 open Tezos
 
-type t
-val spawn : unit -> t
+(* push *)
```

Comment:
This single word in context doesn't signify anything to me. What do you mean here?

Reply:
https://en.wikipedia.org/wiki/HTTP/2_Server_Push

Reply:
Agreed with @d4hines, the single word does not help. I think a full sentence briefly explaining **why** instead of **what** would be better.
What do you think?

Reply:
@EduardoRFS yes that's more or less how @d4hines and I understood it. I'm in favor of adding more documentation than a single word but I won't block the PR over it.

Reply:
It's not a `why` or `what`, nor `documentation` is just a tag, it means a push. If you don't know what is a push this is not the place to explain.

More references on this. 

https://en.wikipedia.org/wiki/Push_technology